api/types.proto

```diff
@@ -771,6 +784,10 @@ message RootCA {
 	// RootRotation contains the new root cert and key we want to rotate to - if this is nil, we are not in the
 	// middle of a root rotation
 	RootRotation root_rotation = 5;
+
+	// LastForcedRotation matches the Cluster Spec's CAConfig's ForceRotation counter.
+	// It indicates when the current CA cert and key were generated (or updated).
+	uint64 last_forced_rotation = 6;
```

Comment:
I don't particularly feel strongly - the name is to make it clearer that it corresponds with the `ForceRotation` value in the config.